include/knowhere/index/index_node.h

```diff
@@ -185,7 +184,7 @@ class IndexNode : public Object {
                 if (same_or_too_far(dist)) {
                     num_consecutive_over_further_bound++;
                     if (num_consecutive_over_further_bound >
-                        std::max(k_min_num_consecutive_over_further_bound, (size_t)(num_next * range_search_level))) {
+                        static_cast<size_t>(std::ceil(num_next * range_search_level))) {
```

Comment:
Just to double-check, the `num_next` will at least be `1` and we at least need 2 consecutive jumps over further bound to stop. Is it intentional?

Reply:
Yes, that's why I use `ceil`.